hbase-client/src/main/java/org/apache/hadoop/hbase/client/AsyncTable.java

```diff
@@ -744,4 +751,10 @@ default CoprocessorServiceBuilder<S, R> toRow(byte[] endKey) {
    */
   <S, R> CoprocessorServiceBuilder<S, R> coprocessorService(Function<RpcChannel, S> stubMaker,
     ServiceCaller<S, R> callable, CoprocessorCallback<R> callback);
+
+  /**
+   * Similar to above. Use when your coprocessor client+endpoint supports partial results.
+   */
+  <S, R> CoprocessorServiceBuilder<S, R> coprocessorService(Function<RpcChannel, S> stubMaker,
```

Comment:
What if we call this method but the endpoint does not support partial results?

Reply:
If the server does not offer partial results, assuming your client is well-written, it is still safe to use this method. Your callback's `getNextCallable()` method will find itself always returning null.